modules/hcp-create-heterogeneous-nodepools.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-create-heterogeneous-nodepools_{context}"]
+= Creating heterogeneous node pools
+
+You can create heterogeneous node pools by using the `NodePool` custom resource (CR).
+
+.Procedure
+
+* To define a `NodePool` CR, create a YAML file similar to the following example:
```

Comment:
Previous 1-step procedures that created a CR used the `<<"EOF" | oc apply -f -` command. This one just creates the yaml. I'm sure it's fine, just calling it out.

Reply:
yes, I this it looks good as is.